ngrinder-frontend/src/js/components/perftest/report/DetailReport.vue

```diff
@@ -61,6 +61,60 @@
                             <th v-text="i18n('perfTest.report.errors')"></th>
                             <td>{{ test.errors | numFormat }}</td>
                         </tr>
+                        <tr>
+                            <th v-text="i18n('perfTest.report.errorRate')"></th>
+                            <td>{{ test.errors /(test.tests + test.errors)*100 |  numFormat('0,0.00') }}
+                                <code class="ml-1">%</code>
+                            </td>
+                        </tr>
+                        <tr>
+                            <th v-text="i18n('perfTest.report.pct25RT')"></th>
+                            <td>{{ test.pct25RT | numFormat  | numFormat('0,0.00') }}
+                                <code class="ml-1">ms</code>
+                            </td>
+                        </tr>
+                        <tr>
+                            <th v-text="i18n('perfTest.report.pct50RT')"></th>
+                            <td>{{ test.pct50RT | numFormat  | numFormat('0,0.00') }}
+                                <code class="ml-1">ms</code>
+                            </td>
+                        </tr>
+                        <tr>
+                            <th v-text="i18n('perfTest.report.pct75RT')"></th>
+                            <td>{{ test.pct75RT | numFormat  | numFormat('0,0.00') }}
+                                <code class="ml-1">ms</code>
+                            </td>
+                        </tr>
+                        <tr>
+                            <th v-text="i18n('perfTest.report.pct90RT')"></th>
+                            <td>{{ test.pct90RT | numFormat  | numFormat('0,0.00') }}
+                                <code class="ml-1">ms</code>
+                            </td>
+                        </tr>
+                        <tr>
+                            <th v-text="i18n('perfTest.report.pct95RT')"></th>
+                            <td>{{ test.pct95RT | numFormat  | numFormat('0,0.00') }}
+                                <code class="ml-1">ms</code>
+                            </td>
+                        </tr>
+                        <tr>
+                            <th v-text="i18n('perfTest.report.pct99RT')"></th>
+                            <td>{{ test.pct99RT | numFormat  | numFormat('0,0.00') }}
+                                <code class="ml-1">ms</code>
+                            </td>
+                        </tr>
+                        <tr>
+                            <th v-text="i18n('perfTest.report.minRT')"></th>
+                            <td>{{ test.minRT | numFormat  | numFormat('0,0.00') }}
+                                <code class="ml-1">ms</code>
+                            </td>
+                        </tr>
+                        <tr>
+                            <th v-text="i18n('perfTest.report.maxRT')"></th>
+                            <td>{{ test.maxRT | numFormat  | numFormat('0,0.00') }}
```

Comment:
Why did you round to decimal by using `numFormat` filter twice?
![image](https://user-images.githubusercontent.com/10591327/197674196-613744ab-b896-4643-be08-e61ae4e79275.png)

The result what you expected is like capture below?
![image](https://user-images.githubusercontent.com/10591327/197674280-bd7c4383-aaa2-44d4-b310-65e5bcef01b3.png)

If so, you have to change `numFormat` filter like this.
```vue
<td>{{ test.maxRT | numFormat('0,0.00') }}
```

Reply:
tks, I will fix it.